docs/TileLink-Diplomacy-Reference/index.rst

```diff
@@ -0,0 +1,29 @@
+TileLink and Diplomacy Reference
+================================
+
+TileLink is the cache coherence and memory protocol used by RocketChip and
+other Chipyard generators. It is how different modules like caches, memories,
+peripherals, and DMA devices communicate with each other.
+
+TileLink is built on top of diplomacy, a framework for exchanging
```

Comment:
Shouldn't TileLink the protocol be independent of diplomacy (or other implementations)? (or isn't that at least the intention of the current revision)

Reply:
True. I guess it should be "RocketChip's TileLink implementation is built on top of diplomacy".